src/main/java/frc/robot/subsystems/vision/VisionSubsystem.java

```diff
@@ -61,23 +69,24 @@ public boolean canSeeAprilTags(int limelightNumber) {
    * @param limelightNumber the number of the limelight
    */
   public void updateLimelightPoseEstimate(int limelightNumber) {
-    if (!canSeeAprilTags(limelightNumber)) {
-      limelightEstimates[limelightNumber] = new PoseEstimate();
-    }
-
-    // double distanceToAprilTags = getLimelightAprilTagDistance(limelightNumber);
-
-    // if (isLargeDiscrepancyBetweenMegaTag1And2(limelightNumber)
-    //     && distanceToAprilTags < VisionConstants.MEGA_TAG_2_DISTANCE_THRESHOLD) {
-    //   limelightEstimates[limelightNumber] = getMegaTag1PoseEstimate(limelightNumber);
-    // } else
-    if (headingRateDegreesPerSecond < VisionConstants.MEGA_TAG_2_MAX_HEADING_RATE) {
-      LimelightHelpers.SetRobotOrientation(
-          getLimelightName(limelightNumber), headingDegrees, 0, 0, 0, 0, 0);
-      limelightEstimates[limelightNumber] = getMegaTag2PoseEstimate(limelightNumber);
-    } else {
-      limelightEstimates[limelightNumber] = getMegaTag1PoseEstimate(limelightNumber);
-    }
+      //Soon to be implemented code:
+    // if (canSeeAprilTags(limelightNumber)) {
+    //   if (isValidPoseEstimate(limelightNumber)) {
+        // double distanceToAprilTags = getLimelightAprilTagDistance(limelightNumber);
+
+        // if (isLargeDiscrepancyBetweenMegaTag1And2(limelightNumber)
+        //     && distanceToAprilTags < VisionConstants.MEGA_TAG_2_DISTANCE_THRESHOLD) {
+        //   limelightEstimates[limelightNumber] = getMegaTag1PoseEstimate(limelightNumber);
+        // } else
+        if (headingRateDegreesPerSecond < VisionConstants.MEGA_TAG_2_MAX_HEADING_RATE) {
+          LimelightHelpers.SetRobotOrientation(
+              getLimelightName(limelightNumber), headingDegrees, 0, 0, 0, 0, 0);
```

Comment:
with the new MT update, idk if we'll have to set the robotOrientation anymore. Could you look into that?

Reply:
yes sir